addons/sys_core/fnc_isPaused.sqf

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+/*
+ * Author: Thymo-
+ * Check if ACRE is paused
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * Paused <BOOL>
+ *
+ * Example:
+ * [] call acre_sys_core_fnc_isPaused
+ *
+ * Public: No
+ */
+
+
+ACRE_IS_PAUSED
```

Comment:
This function is only used in API, we can use the global variable there directly as well.